IFTB.bs

```diff
@@ -0,0 +1,535 @@
+
+### The Compressed Chunk Format (IFTZ) ### {#iftz-format}
+
+A compressed chunk string is the same as an uncompressed chunk string except:
+
+1.  The <code>version</code> is <code>IFTZ</code>.
+2.  All data after the <code>length</code> field is [[!RFC7932|brotli compressed]]
+
+The compressed chunk contains no indication of its (compressed) length, but that length is encoded
+in the <code>chunkOffsetList</code> of the IFTB table.
+
+Browser Behaviors {#iftb-browser-behaviors}
+-------------------------------------------
+
+### First Request ### {#iftb-browser-behaviors-first-request}
+
+1.   The browser loads a font file and identifies it as an IFTB font by its "sfnt" header version (in
+     the uncompressed case) or by its WOFF2 "flavor" (in the compressed case).  These are the first
+     and second 4-byte words in the files respectively.
+2.   In the compressed case, it decodes the WOFF2 into its corresponding OTF (The browser can 
+     optionally isolate the IFTB and cmap tables early in the stream to proceed before the remaining
+     font content downloads.)
+3.   The browser determines the initial font subset description corresponding to the content it needs
+     to render using the font.  (IFTB does not provide subsetting of variable axes, so this is just
+     the sets of codepoints and layout features.)
+4.   The browser maps each codepoint in the font subset description to a GID using the cmap table. It
+     then maps that GID to a chunk index using GIDMap of the IFTB table. That index is added to a
+     "chunk set" (which could be a bitmap or std::vector<bool> indexed by chunk index.
+5.   The browser then look up each layout feature in the font subset description in the IFTB table
+     featureTable. That table maps the initial GID-mapped chunks to higher-indexed feature-specific
+     chunks. If any chunk in the set maps to a feature-specific-chunk the latter is added to the set.
+6.   The browser subtracts the list of already-included chunks recorded in the IFTB ChunkMap from the
+     chunk set.  If no chunks remain, the process skips to step XXX.
+7.   At this point there are two options for retrieving chunk contents:
+
+     Load each needed chunk using individual files:
+     1.  Calculate the hex string for the chunk index
+     2.  Get the chunkFilesURI string from the IFTB table and substitute "$1", "$2", ... with
+          the corresponding hex digit.
+     3.  Combine the relative URI with the URL of the original font to get the URL for the chunk.
+     4.  Request that file via HTTP.
+
+     Load each needed chunk from the range file:
+     1.  Get the chunkRangeFileURI from the string IFTB table.
+     2.  Combine that URI with the URL of the original font to get the URL for the range file.
+     3.  Get the byte range for each chunk from the chunkOffsetTable in the IFTB table.
+     4.  Form a HTTP range request for those byte ranges using the combined URL.
+8.   Decompress each retrieved chunk content.
```

Comment:
Need to add a step here where the chunk's id field is compared to the fonts id and what to do if they don't match.